src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -34,7 +35,8 @@ public class EditCommandParser implements Parser<EditCommand> {
     public EditCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_REWARD, PREFIX_TAG);
+                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL,
+                    PREFIX_BIRTHDAY_MONTH, PREFIX_REWARD, PREFIX_TAG);
```

Comment:
Looks good, for now, we will let them change the birthday month for ease in correcting the user's typo.